src/bundle/Core/Resources/config/routing.yml

```
@@ -26,9 +26,7 @@ services:
         tags:
             - { name: kernel.event_subscriber }
 
-    Ibexa\Core\MVC\Symfony\Component\Serializer\CompoundMatcherNormalizer:
-        tags:
-            - { name: serializer.normalizer }
+    Ibexa\Core\MVC\Symfony\Component\Serializer\CompoundMatcherNormalizer: ~
```

Personally, I think it would be best to explicitly disable autoconfiguration for this service explicitly. My reasoning behind it is that it is very easy to "just enable it" on the declaration file level (as `__defaults`), without realizing the implications that it would cause for this particular service.
```suggestion
    Ibexa\Core\MVC\Symfony\Component\Serializer\CompoundMatcherNormalizer:
        autoconfigure: false
```

---

@Steveb-p Done